server.js

```diff
@@ -31,26 +32,28 @@ app.get('/user', function (req , res) {
 
 // Abdulhameed
 app.post('/login', function (req , res) {
-	console.log("reqbody",req.body)
+
+	// res.redirect('./templates/login.html');
 	db.User.findOne({'username':req.body.username,'password':req.body.password},function (err, data) {
-		if(err){res.send(err)}
-		console.log("data",data)
+		if(err){res.sendStatus(404)}
```

Comment:
Be careful with this. Your error isn't necessarily going to be a 404. 404 is specifically for when the record isn't found. There may be something else failing in your code, in which case, you want to send a 500. Instead, you should send 404 if you don't get any data back. Below, you have a ```if(data!==null)``` conditional. You should have an ```if(data===null)``` conditional instead, which, if true, has the server send a 404. If it is false, you send your normal response.